sdk/digitaltwins/dtdl-parser/review/dtdl-parser.api.md

```diff
@@ -96,7 +98,9 @@ export interface DoubleInfo extends NumericSchemaInfo {
 }
 
 // @public (undocumented)
-export type DtmiResolver = (identifiers: string[]) => Promise<string[] | null>;
+export type DtmiResolver = (dtmis: string, options?: GetModelsOptions) => Promise<{
+    [dtmi: string]: unknown;
```

Comment:
Do we have a general preference between `{ [dtmi: string]: unknown }` and `Record<string, unknown>`? In this particular case IMO the former syntax is a little harder to parse with the promise and union.

Reply:
fair. for now going to add this to the issue created to handle more feedback on the Parser before GA, to get this sorted out fully across the library.

Reply:
I commented on the feedback issue, agreed we can sort this out more later since I don't think it would be considered breaking